crates/dbsp/src/trace/ord/zset_batch.rs

```diff
@@ -31,6 +37,44 @@ pub struct OrdZSet<K, R> {
     pub layer: ColumnLayer<K, R>,
 }
 
+impl<K, R> Encode for OrdZSet<K, R>
```

Comment:
Is this encoding used for secondary storage too?

Reply:
This is the encoding used by the persistence feature. It seems reasonable to adopt the same one. We can change it later if necessary.